### PR TITLE
Template1

### PR DESCRIPTION
**Pull request by PraveenKumar141095**
Added .html files
Added .css files
Update README.md 
Added images folder.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        <div class="wifi">
+        <img src="images/header/Wifi.png" alt="wifi_image">
+        </div>
+        <div class="battery">
+        <img src="images/header/Battery.png" alt="battery">
+        </div>
+      </div>
+      <nav>
+        <div class="nav_logo">
+            <h1>My logo</h1>
```
Comment (franclobo) [inline index.html]: Kindly add `<a>` tag to the logo this is for a better user experience. 👍🏽

File: style.css [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  width: 375px;
+  margin: auto;
+}
+
+#content {
+  width: 375px;
+  height: 10px;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
```
Comment (franclobo) [inline style.css:84]: - [ ] **[OPTIONAL]** Kindly we could delete these icons 👇🏽 they are for reference that it is a mobile version.

image


- [ ] Please 🙏🏽 correct the margins of your design to be similar to the Figma template:

- Your design

image

- Figma template

image

File: style.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  color: #344563;
+  text-align: center;
+}
+
+.socialMedia > h3 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+  color: #7f8cff;
+}
```
Comment (franclobo) [inline style.css]: - Please 🙏🏽 add `font-weight` to look similar to the Figma template 👇🏽 

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    <div class="background">
+        <img src="images/header/header.png" alt="background">
+    </div>
+    <section id="content">
+        <div class="heading">
+            <h2>
+                I'm Praveen 
+                Glad to see 
+                you! 
+            </h2>
```
Comment (franclobo) [inline index.html]: - Kindly add break lines to separate the lines. It is for the best practices of HTML & CSS. 👍🏽 We could check this [information](https://www.w3schools.com/Tags/tag_br.asp).

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Praveen1995-10-14, 

Good job so far!
You did it well, congratulations!! 👏🏽 👏🏽 👏🏽 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Profesional README file ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please 🙏🏽 fix the margins of your design 👇🏽 

image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ghost — APPROVED** [review]
Hi @Praveen1995-10-14 , 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)
## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**